app/controllers/application_controller.rb

```diff
@@ -7,10 +7,13 @@ def create_exporter
     @patient = Patient.find(params[:patient_id])
     @exporter = COVIDHealthCardExporter.new(@patient)
   rescue ActiveRecord::RecordNotFound => e
-    raise e unless params[:format] == 'fhir_json'
-
-    issue = FHIR::OperationOutcome::Issue.new(severity: 'error', code: 'not-found',
-                                              diagnostic: 'Patient does not exist')
-    render json: FHIR::OperationOutcome.new(issue: issue).to_json, status: :not_found and return
+    respond_to do |format|
```

Comment:
The patients controller needs this same handling. Performing a FHIR read on a non-existent patient returns html. Same for immunizations.

Reply:
Roger, I'll add that too